contents/copy/scholarships/guide.md

```diff
@@ -1,3 +1,91 @@
+- This isn't a formal rule, but in Germany, you're not supposed to make too much noise in public transport vehicles. People will look at you weird, and sometimes ask you to tone it down.
+
+## Airports
+
+Berlin has two airports:
+
+- Berlin Tegel (TXL), which lies in the north of Berlin, is the busiest and also likely the one you'll arrive in if you come from outside of Europe. It's a small and fairly hectic airport, and one of its biggest downsides is that the only means of public transport connection is via bus. Hop onto the TXL bus (yes, the bus line is called TXL), and you'll be at the main station or Alexanderplatz in no time. Alternatively, just take a taxi (remember, we'll reimburse you for it).
+- Berlin Schönefeld (SXF), which lies outside of the city limits in the south, is the low-cost and smaller of the two. Fortunately, it has good train connections, with a commuter train connecting the city centre and the airport arriving every 5-10 minutes. Additionally, there's a regional train that takes you directly to the main station. You can take the commuter train directly until Warschauer Straße and then walk to the hotel from there!
+
+## Major Train Stations
+
+If you're arriving from within Europe, chances are that you're arriving by train. Here's a list of the long-distance train stations in Berlin:
+
+- **Hauptbahnhof (Central Station)** — The biggest and most central one. Good connection to the S-Bahn system and the Tram system.
+- **Südkreuz (South Cross)** — Situated at the south of the inner city. Connects to the Ring line as well as the S1 and S2 train lines.
+- **Gesundbrunnen/Nordkreuz (North Cross)** — Situated at the north of the inner city. Connects to the Ring line, the S1 and S2 lines, and the U8 subway line.
+- **Ostbahnhof (East Station)** — The former central station. A stone throw away from the A&O hotel, just take any eastbound S-Bahn line.
+
+# Things about Berlin that you might want to know
+
+- If you would like to explore the city by bike, you can borrow bikes at a lot of shops, as well as with the "nextbike" stands you'll find scattered around the city. Berlin is a mostly bike-friendly city, but that can't be said about some of its inhabitants. Be aware at all times and you should be good, though.
+- Being a capital city, most people in Berlin speak or at least understand English, so there's no need to learn German just for this event.
+- If you need hygiene or sanitary products, most corner shops (colloquially called "Spätis") sell those.
+- Generally, if you need anything late at night, those corner shops are open until after midnight and have a good selection of common products you'd need.
+
+# Schedule Information
+
+We will fill this in as soon as we settle on a detailed schedule! Until then, if you have any questions about the schedule, feel free to contact one of the contacts that are listed above!
```

Comment:
The CSSconf EU schedule is now up here:
https://2018.cssconf.eu/schedule/

(Last-minute changes are always possible, but for now it's final)